test/uchen/tboard/tboard_file.test.cc

```diff
@@ -1,3 +1,45 @@
+#include "uchen/tboard/tboard_file.h"
+
+#include <filesystem>
+
 #include <gtest/gtest.h>
 
-TEST(FileTest, WritesLoss) { FAIL() << "Not implemented!"; }
\ No newline at end of file
+TEST(FileTest, WritesLoss) {
+  auto tboard_file = uchen::tboard::TBoardFile::Open(
+      "events.out.tfevents");
+  if (!tboard_file.ok()) {
```

Comment:
```suggestion
  if (!tboard_file.ok()) {
```
CHECK_TRUE(tboard_file.ok()) << tboard_file.status();

Reply:
I did not find `CHECK_TRUE` in `GTest` API but I found `ASSERT_TRUE` which provides same functionality, used it instead.

Reply:
Yup, thanks! I confused ABSL asserts library and GTest.